src/encryption.rs

```diff
@@ -33,8 +33,8 @@ pub struct EncryptionSettings {
 
     /// Should untrusted devices receive the room key, or should they be
     /// excluded from the conversation.
-    #[wasm_bindgen(js_name = "onlyAllowTrustedDevices")]
-    pub only_allow_trusted_devices: bool,
+    #[wasm_bindgen(js_name = "sharingStrategy")]
+    pub sharing_strategy: CollectStrategy,
```

Comment:
It would be good to update the tests to test reading, and modifying, this.

Reply:
Added test